src/main/java/org/molgenis/vcf/inheritance/genemapper/OmimLineMapper.java

```diff
@@ -0,0 +1,138 @@
+package org.molgenis.vcf.inheritance.genemapper;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.molgenis.vcf.inheritance.genemapper.model.InheritanceMode;
+import org.molgenis.vcf.inheritance.genemapper.model.GeneInheritanceValue;
+import org.molgenis.vcf.inheritance.genemapper.model.Phenotype;
+
+public class OmimLineMapper {
+  public static final String SUBVALUE_SEPARATOR = ";";
+  public static final String VALUE_SEPARATOR = ",";
+  public static final int COMMENTS_INDEX = 12;
+  public static final int GENE_INDEX = 6;
+
+  private OmimLineMapper() {}
+
+  static List<GeneInheritanceValue> parseOmimLine(String line) {
+    List<GeneInheritanceValue> geneInheritanceValues = new ArrayList<>();
+    String[] split = line.split("\t", -1);
```

Comment:
how about using opencsv similar to https://github.com/molgenis/mvl-vcf-converter for reading/writing tabular data? makes things a bit more robust and readable.